src/app/modules/office/components/offices/offices.component.ts

```diff
@@ -9,28 +15,36 @@ import { OfficeModel } from '../../../../shared/models/office.model';
 export class OfficesComponent implements OnInit {
   public isOpen = false;
   public activeEditId = '';
-  public offices = [];
-  constructor() { }
-  public ngOnInit(): void {}
+  public offices$: Observable<OfficeModel[]>;
+  public officesCount$: Observable<number>;
+  constructor(
+    private store: Store<OfficesState[]>,
+  ) { }
+
+  public ngOnInit(): void {
+    this.offices$ = this.store.select(selectAllOffices);
+    this.officesCount$ = this.store.select(selectOfficesCount);
+    this.store.dispatch(loadOffices());
+    this.offices$.subscribe(() => {
+      this.handleForm(false);
+    });
+  }
```

Comment:
hm, I don't know about this.offices$ and this.offices$.subscribe...if it the best variant..mybe this.store.select(selectAllOffices).subscribe() and inside it handleForm and set office variable